enzyme.config.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
```

Comment:
Not sure if you still need this file since you have the same thing on client/components/MapText.test.jsx

Reply:
Thanks for catching this. I updated the MapText.test.jsx file to remove the enzyme configurations. The enzyme.config.js file is actually needed for the jest.config.js file.